src/pages/starter-kit/checkout/shipping-reference.md

```diff
@@ -48,18 +67,56 @@ curl --request POST \
 }'
 ```
 
-Example of the update request:
+#### Example response
+
+```json
+{
+  "success": true,
+  "message": {
+    "id": 1,
+    "code": "DPS",
+    "title": "Demo Postal Service",
+    "stores": ["default"],
+    "countries": ["US", "CA"],
+    "active": true,
+    "sort_order": 10,
+    "tracking_available": true,
+    "shipping_labels_available": true
+  }
+}
+```
+
+### Update an existing OOPE shipping carrier
+
+The PUT `/V1/oope_shipping_carrier` updates an out-of-process shipping carrier in the Adobe Commerce instance.
+
+**Payload parameters:**
+
+| Parameter                   | Type    | Required | Description                                                              |
+| --------------------------- | ------- | -------- | ------------------------------------------------------------------------ |
+| `code`                      | String  | Yes      | Unique identifier for the shipping carrier.                              |
+| `title`                     | String  | Yes      | Display name of the shipping carrier.                                    |
+| `stores`                    | Array   | No       | List of store codes where the shipping carrier is available.             |
+| `countries`                 | Array   | No       | List of countries where the shipping carrier is available.               |
+| `active`                    | Boolean | No       | Status indicating if the shipping carrier is active.                     |
+| `sort_order`                | Integer | No       | The sort order of shipping carriers.                                     |
+| `tracking_available`        | Boolean | No       | Status indicating if the shipping carrier has available tracking.        |
+| `shipping_labels_available` | Boolean | No       | Status indicating if the shipping carrier has available shipping labels. |
+
+<CodeBlock slots="heading, code" repeat="2" languages="bash, json" />
+
+#### Example request
 
 ```bash
 curl --request PUT \
---url <ADOBE_COMMERCE_URL>/rest/all/V1/oope_shipping_carrier \
+--url <ADOBE_COMMERCE_API_URL>/V1/oope_shipping_carrier \
 --header 'Authorization: Bearer <TOKEN>' \
 --header 'Content-Type: application/json' \
 --data '{
 "carrier": {
-"code": "DEPS",
-"title": "Demo postal service",
-"stores": ["default"],
+"code": "DPS",
+"title": "Detroit Postal Service"
```

Comment:
Either revert this or change other instances.

Reply:
@jcuerdo this should be "Demo" or the other examples should be changed to match.

```suggestion
"title": "Demo Postal Service"
```